Improve shared layout for Wmma's operands

leeliu103 opened the pull request:
1. Enable swizzling for Wmma's B operand
2. Refine the formula for vectorSize/perPhase/maxPhase

zhanglx13 reviewed (approved):
This one can go upstream directly

leeliu103 committed:
Improve shared layout for Wmma's operands

1. Enable swizzling for Wmma's B operand
2. Refine the formula for vectorSize/perPhase/maxPhase
